test/fastly-rails/configuration_test.rb

```diff
@@ -65,6 +65,26 @@
     end
   end
 
+  describe 'cache_headers_enabled?' do
+    before do
+      configuration.cache_headers_enabled = true
+    end
+
+    it 'is true' do
+      assert_equal true, configuration.cache_headers_enabled?
+    end
+  end
+
+  describe 'cache_headers_enabled?' do
+    before do
+      configuration.purging_enabled = true
```

Comment:
This is the default behavior. Could you also add tests for the negative? (same for `cache_headers_enabled` please)